Cesium.Test.Framework/ParserTestBase.cs

```diff
@@ -10,24 +10,7 @@ public abstract class ParserTestBase : VerifyTestBase
 {
     protected static string? GetErrorString<T>(ParseResult<T> result)
     {
-        if (!result.IsError) return null;
-
-        var errorMessage = new StringBuilder();
-        var err = result.Error;
-        foreach (var element in err.Elements.Values)
-        {
-            errorMessage.AppendLine($"expected {string.Join(" or ", element.Expected)} while parsing {element.Context}");
-        }
-
-        var got = err.Got switch
-        {
-            null => "end of input",
-            IToken<CTokenType> t => $"{t.Kind}: {t.Text}",
-            var o => o.ToString()
-        };
-        errorMessage.AppendLine($" but got {got}");
-
-        return errorMessage.ToString();
+        return result.GetErrorString();
```

Comment:
_(Minor)_ I guess we can eliminate this method and just call the extension everywhere?

Reply:
Yes. Obviously we can. I just do not think this should be part or this PR. Better handle that kind of refactoring in follow up PR IMO